js/src/components/ConnectGoogleSearchConsole.js

```diff
@@ -317,19 +330,29 @@ class ConnectGoogleSearchConsole extends React.Component {
 			return (
 				<div>
 					{profileSelectBox}
-					<RaisedButton label="Reauthenticate with Google" onClick={this.clearAuthCode.bind( this )} />
+					<RaisedButton
+						label={this.props.translate( "Reauthenticate with Google" )}
+						onClick={this.clearAuthCode.bind( this )} />
 					{loader}
 				</div>
 			);
 		}
-
+		/* Translators: %s expands to Yoast SEO  */
```

Comment:
Please place this translator comment directly above the string to translate to make sure the it will be parsed.